### DISCUSSION TITLE
Configuration on CI/CD

### DISCUSSION THREAD

**Discussion by SimonasB88** [FRSOURCE/cypress-plugin-visual-regression-diff#307] [Q&A] [answered]
Hello,

I'm new to this nice framework. Wanted to ask, is the snapshot updated every CI run, and stored in the ubuntu of the CI machine? Or the snapshot is overrided by newer snapshot each CI run? 

Thanks.

**Comment by astrodomas** [comment, marked as answer]
Refer to configuration [docs](https://github.com/FRSOURCE/cypress-plugin-visual-regression-diff?tab=readme-ov-file#configuration)

Your question is not really clear. You should declare what you want to achieve.
But basically on each CI run you will run the test and it will `create` (respectively it will create diff images or generate a new one if there is no image) or `override` (update the existing image) files if visual regression is detected. However, the regenerated or diff images will be stored inside the CI container which - after the CI process finishes - will be destroyed.

So, let's say you want to persist the images generated in the CI. You could commit and push changes from the container to your Pull Request you made and see the outcome.

**Comment by FRSgit** [comment]
Closing this discussion as answered. Thanks @astrodomas and @SimonasB88!
